pycsob/utils.py

```diff
@@ -46,6 +46,8 @@ def mk_msg_for_sign(payload):
         for one in payload['cart']:
             cart_msg.extend(one.values())
         payload['cart'] = '|'.join(map(str_or_jsbool, cart_msg))
+    if 'customer' in payload and payload['customer'] not in conf.EMPTY_VALUES:
```

Comment:
How about this:
```suggestion
    if payload.get('customer') not in conf.EMPTY_VALUES:
```
? :)

Reply:
I followed the condition structure as above, but I could use this pattern.

Reply:
Ah, I didn't see that. 👍